src/Components/Publishing/Sections/ImageCollection.tsx

```diff
@@ -53,6 +56,7 @@ class ImageCollectionComponent extends React.PureComponent<ImageCollectionProps,
           <Image
             image={image}
             sectionLayout={sectionLayout}
+            layout={articleLayout}
```

Comment:
This is where we pass it through. `Image` now knows that this has a `layout="news"` so it won't render that `IconExpand`.